packages/simple-types/test/dashboard.test.ts

```diff
@@ -194,4 +225,76 @@ describe("Module `dashboard`: manages the creation and deployment of dashboard i
       expect(dsInfos[1].references[0]).toEqual("map0");
     });
   });
+
+  describe("_templatizeByDatasource", () => {
+    it("ignores supplied objs if it is not defined", () => {
+      const updatedList = dashboard._templatizeByDatasource(null, null);
+      expect(updatedList).toBeNull();
+    });
+  });
+
+  describe("_getDatasourceInfo", () => {
```

Comment:
same here?

Reply:
@jmhauck, these look like merge errors that I made; I'll fix; thanks